manifests/params.pp

```diff
@@ -13,20 +13,31 @@
         $service_file_location   = '/usr/lib/systemd/system/jira.service'
         $service_file_template   = 'jira/jira.service.erb'
         $service_lockfile        = '/var/lock/subsys/jira'
+        $service_systemd         = true
```

Comment:
how about making this variable `$service_provider = 'systemd'` instead of a boolean?